UI/src/view/initialPopupView.js

```diff
@@ -0,0 +1,215 @@
+window.InitialPopupView = function(backend, location, userRole, places, assetTypeConfig, defaultAsset, municipality) {
+
+  var options = {
+    confirmButton: 'Tallenna',
+    cancelButton: 'Peruuta',
+    successCallback: function () {
+      var selectedLocation = $('#location').val();
+      var selectedAssetType = parseInt($('#assetType').val(), 10);
+      var defaultParameters = {lon: null, lat: null, zoom: null, assetType: selectedAssetType, municipalityId: null, elyId: null};
+
+      if (selectedLocation === "currentLocation") {
+        defaultParameters.lon = location.lon;
+        defaultParameters.lat = location.lat;
+        defaultParameters.zoom = location.zoom;
+      }
+      else if (selectedLocation === "suomi") {
+        defaultParameters.lon = 390000;
+        defaultParameters.lat = 6900000;
+        defaultParameters.zoom = 2;
+      }
+      else {
+        if (userRole !== "busStopMaintainer") {
```

Comment:
Could use something like this `if (!isElyMaintainer())` by using the respective function in authorizationPolicy.js